sdk/core/Azure.Core/src/Pipeline/HttpClientTransport.cs

```diff
@@ -613,5 +614,9 @@ private static void SetPropertiesOrOptions<T>(HttpRequestMessage httpRequest, st
             httpRequest.Properties[name] = value;
 #endif
         }
+
+        private static bool UseCookies() => AppContextSwitchHelper.GetConfigValue(
+            "Azure.Core.Pipeline.HttpClientTransport.EnableCookies",
+            "AZURE_CORE_HTTPCLIENT_ENABLE_COOKIES");
```

Comment:
Should we add this config info to the changelog?

Reply:
I think we don't want to advertise this.